modules/tensor_mechanics/include/materials/ADRadialReturnStressUpdate.h

```diff
@@ -65,12 +65,12 @@ class ADRadialReturnStressUpdate : public ADStressUpdateBase<compute_stage>,
   virtual Real computeReferenceResidual(const ADReal & effective_trial_stress,
                                         const ADReal & scalar_effective_inelastic_strain) override;
 
-  virtual Real minimumPermissibleValue(const ADReal & /*effective_trial_stress*/) const override
+  virtual ADReal minimumPermissibleValue(const ADReal & /*effective_trial_stress*/) const override
```

Comment:
I still don't get what I should set the jacobian error for the ad_power_law_creep tests...but I wanted to see what you thought of these changes @dschwen? The min/max permissible values _do_ touch the residual, but only check checking bounds or ranges, and are likely overwritten with full ADReals once converged...but it seems simple enough to just have it fully implemented.

Reply:
Reminder to @dschwen. I think that these should probably be ADReals since they are part of the chain of incremental changes to the scalar value being solved by the newton loop.

Reply:
Yeah, you are right!